packages/integrations/image/README.md

```diff
@@ -333,13 +348,14 @@ See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/size
 
 **Type:** `number[]`<br>
 **Required:** `true`
+
 </p>
 
 The list of sizes that should be built for responsive images. This is combined with `aspectRatio` to calculate the final dimensions of each built image.
 
 ```astro
 // Builds three images: 400x400, 800x800, and 1200x1200
-<Picture src={...} widths={[400, 800, 1200]} aspectRatio="1:1" alt="descriptive text" />
+<Picture src={/* ... */} widths={[400, 800, 1200]} aspectRatio="1:1" alt="descriptive text" />
```

Comment:
Need a small tweak here so prettier-plugin-astro can parse this.

Reply:
Could we just pass some variable here?

```suggestion
<Picture src={img} widths={[400, 800, 1200]} aspectRatio="1:1" alt="descriptive text" />
```